src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx

```diff
@@ -49,32 +49,38 @@ const useStyles = makeStyles(() => ss);
  * @returns {JSX.Element} - Drawer component
  */
 export function DrawerLayoutComponent(props: {
-  filterSelection: JSX.Element;
-  displayFeatureSelection: JSX.Element;
-  isOpen: boolean;
-  drawerWidth: number;
-  drawerTitle: string;
+    filterSelection: JSX.Element;
+    displayFeatureSelection: JSX.Element;
+    isOpen: boolean;
+    drawerWidth: number;
+    drawerTitle: string;
 }): JSX.Element {
-  const drawerWidthSting = (props.drawerWidth as unknown) as string;
-  const classes = useStyles({
-    drawerWidth: drawerWidthSting
-  });
+    const drawerWidthSting = (props.drawerWidth as unknown) as string;
+    const classes = useStyles({
+        drawerWidth: drawerWidthSting,
+    });
 
-  return (
-    <Drawer
-      className={classes.drawer}
-      variant="persistent"
-      anchor="left"
-      open={props.isOpen}
-      classes={{
-        paper: classes.drawerPaper
-      }}
-    >
-      <div className={classes.drawerContainer}>
-        <p css={filterHeadingStyle}>{props.drawerTitle}</p>
-        {props.filterSelection}
-        {props.displayFeatureSelection}
-      </div>
-    </Drawer>
-  );
+    return (
+        <Drawer
+            className={classes.drawer}
+            variant="persistent"
+            anchor="left"
+            open={props.isOpen}
+            classes={{
+                paper: classes.drawerPaper,
+            }}
+            sx={{ 
```

Comment:
I don't understand this, looks like a hack. The cause of the problem might be elsewhere.
But for a fix it is ok.

Reply:
No this is not a hack. This is new by the update to material ui 5. In the long run the whole css will be changed to it.

Reply:
And what does sx mean?
I think they want to make their components more customizable. Or what is the reason behind it?